src/main/java/dk/cngroup/lentils/controller/CypherController.java

```diff
@@ -108,4 +112,14 @@ public String deleteCypher(@PathVariable("cypherId") final Long cypherId) {
         cypherService.deleteById(cypherId);
         return REDIRECT_CYPHER_LIST;
     }
+
+    private void mapOrganizersToCypher(final CypherFormDTO command, final Cypher cypher) {
+        cypher.setOrganizers(userService.getOrganizersByIds(command.getOrganizers()));
+    }
+
+    private void mapOrganizersIdsToCypherFormDTO(final Cypher cypher, final CypherFormDTO cypherFormDto) {
+        cypherFormDto.setOrganizers(cypher.getOrganizers().stream()
+                .map(User::getUserId)
+                .collect(Collectors.toList()));
+    }
```

Comment:
Asi se mi nelibi, ze `mapOrganizersToCypher ` a `mapOrganizersIdsToCypherFormDTO ` jsou metody kontroleru. Nemelo by to byt minimalne v service a nebo v nejak classe, ktera to bude implementovat? Co ty na to @crlikcngroup ?